api/tests/prescription/learner-management/unit/domain/models/OrganizationLearnerImportFormat_test.js

```diff
@@ -166,13 +184,32 @@ describe('Unit | Models | OrganizationLearnerImportFormat', function () {
         organizationLearnerImportFormatPayload,
       );
       expect(organizationLearnerImportFormat.orderedFilterableColumns).to.deep.equal([
-        { key: 3, name: IMPORT_KEY_FIELD.COMMON_DIVISION, position: 1 },
-        { key: 4, name: IMPORT_KEY_FIELD.COMMON_BIRTHDATE, position: 2 },
+        { name: IMPORT_KEY_FIELD.COMMON_BIRTHDATE, position: 1 },
+        { name: IMPORT_KEY_FIELD.COMMON_DIVISION, position: 2 },
       ]);
     });
 
     it('should return empty when filterableColumns is not defined', function () {
-      delete organizationLearnerImportFormatPayload.config.filterableColumns;
+      organizationLearnerImportFormatPayload.config.headers = [
```

Comment:
**question**: est ce qu'on pourrait pas mettre ça dans un beforEach ?

Reply:
là, je surcharge la config intial du haut afin de ne pas avoir les données qui ne me sert pas. seulement dans ce it.

ajouter un describe, un beforeEach , juste pour ce it ?  il se repète dans des it. Mais qui sont dans d'autre contexte.

Reply:
j'avais l'impresssion qu'on repetait la surcharge sur les 3 tests suivants